### DISCUSSION TITLE
Dynamic Column Widths

### DISCUSSION THREAD

**Discussion by mpetersheim** [Budibase/budibase#12599] [Help] [answered]
Is it possible to configure the Budibase table in such a way that each column expands to precisely the width of its widest content? I'm new to Budibase, and from what I've seen columns are all the same width, which ends up giving some columns unnecessary padding and other columns getting their content truncated with ellipses (e.g. "sample val...").  
   I just want my table to have each column display all its content in minimal width, with a horizontal scrollbar if content doesn't fit. Currently seems like it can't do a scrollbar unless you set a column to some static width that makes it not fit.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @mpetersheim,

Currently this isn't something which is possible natively in Budibase as the width column isn't something which is bindable. I would suggest creating a feature request [here](https://github.com/Budibase/budibase/issues/new/choose) as i don't believe this would be something thats extremely difficult to do.

Additionally you can look into custom plugins e.g. the [super table](https://github.com/poirazis/bb-component-SuperTable) that might already have this functionality.

I hope this helps.

**Reply by mpetersheim** [reply]
Okay, thanks for the response.
